sipa,2016-11-15T18:16:53Z,"utACK ea83d0098a2eb29a3b1398f3ecf907f7eece1eee\n\n(I wouldn't classify this as trivial though, it actually changes the executed code).\n",https://github.com/bitcoin/bitcoin/pull/9165#issuecomment-260721505,260721505,
instagibbs,2016-11-15T18:20:12Z,"True, which is why I made sure to run the tests. Changed.\n",https://github.com/bitcoin/bitcoin/pull/9165#issuecomment-260722476,260722476,
mrbandrews,2016-11-17T18:43:54Z,"If you were inclined, you could add the following test at line 87 of wallet.py:\nassert_raises_message(JSONRPCException, ""requires a transaction fee"", self.nodes[2].sendtoaddress, self.nodes[2].getnewaddress(), 20.99999999)\nOtherwise, ACK\n",https://github.com/bitcoin/bitcoin/pull/9165#issuecomment-261332056,261332056,
morcos,2016-11-28T16:46:38Z,"utACK\n",https://github.com/bitcoin/bitcoin/pull/9165#issuecomment-263323827,263323827,
gmaxwell,2016-12-04T20:01:57Z,utACK,https://github.com/bitcoin/bitcoin/pull/9165#issuecomment-264726964,264726964,
jonasschnelli,2016-12-04T20:05:17Z,"`CWalletTx` uses a cache for the `GetAvailableCredit()`, but right, `GetBalance` should only be called if really necessary because of the loop under the lock through mapWallet and the `isTrusted()` calls.\n\nutACK ea83d0098a2eb29a3b1398f3ecf907f7eece1eee",https://github.com/bitcoin/bitcoin/pull/9165#issuecomment-264727165,264727165,
